README.md

```diff
@@ -19,17 +19,34 @@ Prerequisites
 Building / Installation
 -----------------------
 
+### Building from the repository
+
+If building from a source release package, e.g. the tarball shipped from
+<https://github.com/t-brown/mcds/releases>, skip to the next section.
+
 `mcds` relies on the GNU build system
 [autoconf](https://www.gnu.org/software/autoconf/) and
-[automake](https://www.gnu.org/software/automake/).
+[automake](https://www.gnu.org/software/automake/) with
+[gnulib](https://www.gnu.org/software/gnulib/).
```

Comment:
Why do we need `gnulib`? I need to read up on it, and test this PR on BSD systems.